core/templates/dev/head/exploration_editor/EditorServices.js

```diff
@@ -700,6 +758,58 @@ oppia.factory('explorationStatesService', [
         return (
           validatorsService.isValidStateName(newStateName, showWarnings));
       },
+      getStateContentMemento: function(stateName) {
+        return getStatePropertyMemento(stateName, 'content');
+      },
+      saveStateContent: function(stateName, newContent) {
+        saveStateProperty(stateName, 'content', newContent);
+      },
+      getStateParamChangesMemento: function(stateName) {
+        return getStatePropertyMemento(stateName, 'param_changes');
+      },
+      saveStateParamChanges: function(stateName, newParamChanges) {
```

Comment:
Not totally straightforward -- it needs a fair amount of setup, and I'm concerned about the additional review round, so I would prefer to defer it in this one. Many of these are covered by protractor tests, though, and I tested the others manually.